src/python/pants/util/docutil.py

```diff
@@ -16,8 +16,15 @@ def terminal_width(*, fallback: int = 96, padding: int = 2) -> int:
     return shutil.get_terminal_size(fallback=(fallback, 24)).columns - padding
 
 
-def doc_url(slug: str) -> str:
-    return f"https://www.pantsbuild.org/v{MAJOR_MINOR}/docs/{slug}"
+def doc_url(path: str, versioned: bool = True) -> str:
```

Comment:
Alternatively, we could determine if it's versioned from the path.
- if the path starts with "docs" or "reference" it's versioned
- we could build a convention that paths that start with a slash are unversioned

Not saying these are better, just wanted to offer some alternate solutions in case you like one

Reply:
I like the first one; good idea.